sds_data_manager/constructs/database_construct.py

```diff
@@ -155,3 +156,50 @@ def __init__(
         # Add an explicit dependency on the RDS instance because we need the secret
         # populated with the DB credentials before we can create the schema.
         db_custom_resource.node.add_dependency(db)
+
+    def add_synchronizer(self, code, layers, bucket_name, vpc):
+        """Add synchronizer lambda.
+
+        Parameters
+        ----------
+        code : lambda_.Code
+            Lambda code bundle to create the initial DB schema.
+        layers : list
+            List of Lambda layers to attach to the lambda function.
+        bucket_name : str
+            S3 bucket name.
+        vpc : ec2.Vpc
+            Virtual private cloud that this lambda should be placed in.
+        """
+        synchronizer_lambda = lambda_.Function(
+            self,
+            id="db-synchronizer",
+            function_name="db-synchronizer",
+            code=code,
+            handler="SDSCode.database.synchronizer.lambda_handler",
+            runtime=lambda_.Runtime.PYTHON_3_12,
+            timeout=cdk.Duration.seconds(180),
```

Comment:
Is this enough time for what you are doing? Should it be set to the max (15 min)?

Reply:
I sure hope we can synchronizer within 3 minutes. If it is taking 15 minutes to go through we are going to have trouble I think. I'm always a fan of starting smaller and running up against the limit rather than making it too large initially.